core/server/proxy/src/main/java/alluxio/proxy/StreamsRestServiceHandler.java

```diff
@@ -70,6 +71,7 @@ public StreamsRestServiceHandler(@Context ServletContext context) {
   @POST
   @Path(ID_PARAM + CLOSE)
   @ReturnType("java.lang.Void")
+  @Nullable
```

Comment:
how can these methods return null? It looks like `RestUtils.call` should never return `null`